### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ACHultman**

### CODE REVIEW

**Comment by ACHultman** [comment]
initial tests + setup by @bmazerolle

File: src/components/UI/organisms/SignUpUser.js [hunk 1]
```diff
@@ -1,67 +1,94 @@
...
+            <Form.Control
+              id="lastname"
+              type="text"
+              onChange={(e) => setLastName(e.target.value)}
+            />
+          </Form.Group>
+          <Form.Group className="mb-3">
+            <Form.Label htmlFor="username">USERNAME</Form.Label>
+            <Form.Control
+              id="username"
```
Comment (ACHultman) [inline src/components/UI/organisms/SignUpUser.js:62]: added htmlFor and ids on here and LoginUser

File: src/components/pages/AdminPanel.js [hunk 2]
```diff
@@ -1,70 +1,93 @@
...
+    enableAccount(username, user.jwt).then((resp) => {
+      console.log("User enabled");
+      // Update the list of users
+      setUsers((users) => {
+        return users.map((user) => {
+          if (user.username === username) {
+            user.enabled = true;
+          }
+          return user;
+        });
```
Comment (ACHultman) [inline src/components/pages/AdminPanel.js:32]: added basic feedback in table when enabling user
(user now shows as enabled)
this is not yet received from BE though

File: src/components/pages/Resources.js [hunk 3]
```diff
@@ -1,89 +1,123 @@
...
-					setFiles(resp)});
-		
+const Resources = (props) => {
+  const [selectedFile, setSelectedFile] = useState();
+  const [isFilePicked, setIsFilePicked] = useState(false);
+  const { user } = useContext(UserContext);
+  const [listOfFiles, setFiles] = useState([]);
+  const [dataLoaded, setDataLoaded] = useState(false);
+  const [content, setContent] = useState("");
+  const [errorMessage, setErrorMessage] = useState("");
```
Comment (ACHultman) [inline src/components/pages/Resources.js:19]: added error message and handling in reqs

**Review by bmazerolle — APPROVED** [review]
Looks great

**Pull request merged**